docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md

```diff
@@ -36,7 +36,14 @@ var driver = new FirefoxDriver(options);
 # We don't have a Ruby code sample yet -  Help us out and raise a PR
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+import * as firefox from 'selenium-webdriver/firefox';
```

Comment:
Hi @natanportilho,

i was unable to test the code sample, i faced the below issue

import * as firefox from 'selenium-webdriver/firefox';
       ^

SyntaxError: Unexpected token *
    at Module._compile (internal/modules/cjs/loader.js:723:23)

but placing below line worked for me

const firefox = require('selenium-webdriver/firefox');

Reply:
Hi @Harsha509,

Thanks for your review, I've applied these changes.